tests/cpp_unit_tests/CMakeLists.txt

```diff
@@ -8,29 +8,29 @@ find_package(doctest REQUIRED)
 find_package(nlohmann_json CONFIG REQUIRED)
 
 set(PROJECT_SOURCES
-  	test_power_grid_model.cpp
- 	test_main_model_se.cpp
- 	test_main_model.cpp
+	test_power_grid_model.cpp
+	test_main_model_se.cpp
+	test_main_model.cpp
 	test_main_model_static.cpp
- 	test_three_phase_tensor.cpp
- 	test_node.cpp
- 	test_line.cpp
- 	test_link.cpp
- 	test_load_gen.cpp
- 	test_source.cpp
- 	test_shunt.cpp
- 	test_transformer.cpp
-   	test_sparse_lu_solver.cpp
-  	test_y_bus.cpp
- 	test_math_solver.cpp
- 	test_topology.cpp
- 	test_container.cpp
- 	test_sparse_mapping.cpp
- 	test_meta_data_generation.cpp
- 	test_voltage_sensor.cpp
- 	test_power_sensor.cpp
+	test_three_phase_tensor.cpp
+	test_node.cpp
+	test_line.cpp
+	test_link.cpp
+	test_load_gen.cpp
+	test_source.cpp
+	test_shunt.cpp
+	test_transformer.cpp
+	test_sparse_lu_solver.cpp
+	test_y_bus.cpp
+	test_math_solver.cpp
+	test_topology.cpp
+	test_container.cpp
+	test_sparse_mapping.cpp
+	test_meta_data_generation.cpp
+	test_voltage_sensor.cpp
+	test_power_sensor.cpp
 	test_validation.cpp
-    test_three_winding_transformer.cpp
+	test_three_winding_transformer.cpp
```

Comment:
maybe you can use a `glob` to just get all `cpp` files in this folder.

Reply:
Yes, we could do that, but I like to disable test source files while developing, which makes the building process much faster.